geo/src/algorithm/affine_ops.rs

```diff
@@ -300,12 +313,37 @@ impl<T: CoordNum> AffineTransform<T> {
     /// The argument order matches that of the affine transform matrix:
     ///```ignore
     /// [[a, b, xoff],
-    /// [d, e, yoff],
-    /// [0, 0, 1]] <-- not part of the input arguments
+    ///  [d, e, yoff],
+    ///  [0, 0, 1]] <-- not part of the input arguments
     /// ```
     pub fn new(a: T, b: T, xoff: T, d: T, e: T, yoff: T) -> Self {
         Self([[a, b, xoff], [d, e, yoff], [T::zero(), T::zero(), T::one()]])
     }
+
+    /// Get the x-resolution value.
+    pub fn a(&self) -> &T {
```

Comment:
CoordNum implements `Copy`, so is it better to return `T` here instead?

Reply:
Yeah, finding out that I need to deref or call `.to_owned()` on the value in order to use it elsewhere. Maybe it is better to return a cloned copy?

Reply:
since T is copy, you should be able to just remove the `&` I think